python/numpy-distutils.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+- id: numpy-distutils
+  patterns:
+    - pattern-either: 
+      - pattern: |
+          np.distutils
```

Comment:
I would turn "np" into "numpy" to generalize across different forms of the library name.